fix: resolve plugins from npm packages correctly (#1749)
Redocly/redocly-cli: .changeset/strange-starfishes-press.md

---
"@redocly/openapi-core": patch
"@redocly/cli": patch
---

> Fixed a bug where the plugins in external NPM packages could not be resolved if the CLI package was installed globally.

Reviewer: ```suggestion
Fixed an issue where the plugins in external NPM packages could not be resolved if the CLI package was installed globally.
```

Author: Applied the suggestion
